extensions/common/crypto/jwt-verifiable-credentials/src/main/java/org/eclipse/edc/verifiablecredentials/jwt/JwtPresentationVerifier.java

```diff
@@ -101,12 +120,12 @@ public Result<Void> verifyPresentation(String serializedJwt) {
 
             if (vcTokens.isEmpty()) {
                 // todo: this is allowed by the spec, but it is semantic nonsense. Should we return failure or not?
-                return Result.failure("No credential found in presentation.");
+                return Result.success();
             }
 
             // every VC is represented as another JWT, so we verify all of them
             for (String token : vcTokens) {
```

Comment:
According to the PR title, we now support a JWT presentation containing VC either as JWT of LDP so the notation `vcTokens` is misleading here, as it tends to indicate that only JWT VC can exist here.

Reply:
you're right.